docs/src/modules/components/ApiPage.tsx

```diff
@@ -272,9 +260,11 @@ export default function ApiPage(props: ApiPageProps) {
           </React.Fragment>
         ) : null}
         <PropertiesSection
-          properties={componentProps}
-          propertiesDescriptions={propDescriptions}
-          componentName={pageContent.name}
+          properties={propsApiProcessor({
```

Comment:
This is the new way of defining API pages, such that if someone need to pass other kind of data, they can do it

The previous one should still work the time of the migration